### DISCUSSION TITLE
zh/docs/provider/oauth/google/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2332] [Docs comments]
# zh/docs/provider/oauth/google/

Add the Google OAuth provider to your application

https://casdoor.org/zh/docs/provider/oauth/google/

**Comment by Arvin8613** [comment]
配置好Google Auth 2.0登录后回跳报错，如何解决呢？
Post "https://accounts.google.com/o/oauth2/token": dial tcp 46.82.174.69:443: i/o timeout

**Reply by hsluoyz** [reply]
@Arvin8613 you need to make sure Casdoor server is not blocked to access Google, you can set socks5Proxy at: https://casdoor.org/docs/basic/server-installation/#via-ini-file:~:text=when%20logging%20in.-,socks5Proxy,-is%20the%20SOCKS

**Reply by Arvin8613** [reply]
That helps me a lot. Thx! :)

**Comment by yutingzhao1991** [comment]
"Casdoor的回调URL" 是什么？

**Reply by hsluoyz** [reply]
@yutingzhao1991 Casdoor's callback URL is: `/callback`, see docs: https://casdoor.org/docs/how-to-connect/vue-sdk/#initializing-the-sdk
